projects/ad9361/src.mk

```diff
@@ -35,7 +35,8 @@ SRCS += $(PLATFORM_DRIVERS)/uart.c					\
 	$(NO-OS)/util/list.c						\
 	$(NO-OS)/iio/iio_ad9361/iio_ad9361.c				\
 	$(NO-OS)/iio/iio_axi_adc/iio_axi_adc.c				\
-	$(NO-OS)/iio/iio_axi_dac/iio_axi_dac.c
```

Comment:
I think this change should go in the previous commit. A commit shouldn't brake anything, for example if I checkout the previous commit I'am expecting it to run, but it will not without this commit.
Maybe a better title for the previous commit will be 'Use iio_app' instead of 'work iio_app'